lib/standards/aria-roles.js

```diff
@@ -85,7 +85,7 @@ const ariaRoles = {
 	},
 	combobox: {
 		type: 'composite',
-		requiredOwned: ['textbox', 'listbox', 'tree', 'grid', 'dialog'],
+		requiredOwned: ['listbox', 'tree', 'grid', 'dialog', 'textbox'],
```

Comment:
This was to allow the tests to pass when the order of the required was different. Silly, but we don't have an easy way to check for "equal but order doesn't matter" in chai.